### PR TITLE
Update WebGL 2 spec to add a D3D11 limitation.

### PR DESCRIPTION
**Pull request by zhenyao**

### CODE REVIEW

**Comment by zhenyao** [comment]
@vonture PTAL
@kenrussell @jdashg @grorg @RafaelCintron FYI

File: specs/latest/2.0/index.html [hunk 1]
```diff
@@ -1110,6 +1110,8 @@ <h2 class="no-toc">Table of contents</h2>
         <p>Returns <code>FRAMEBUFFER_UNSUPPORTED</code> if depth and stencil attachments, if present, are not the same image. See <a href="#FBO_ATTACHMENTS">Framebuffer Object Attachments</a> for detailed discussion.</p>
         <p>Returns <code>FRAMEBUFFER_INCOMPLETE_MULTISAMPLE</code> if the values of <code>RENDERBUFFER_SAMPLES</code> are different among attached renderbuffers, or are non-zero if the attached images are a mix of renderbuffers and textures.</p>
         <p>Returns <code>FRAMEBUFFER_INCOMPLETE_DIMENSIONS</code> if attached images have different width, height, and depth (for 3D textures) or array size (for 2D array textures). See <a href="#INCOMPLETE_DIMENSIONS">checkFramebufferStatus may return FRAMEBUFFER_INCOMPLETE_DIMENSIONS</a>.</p>
+        <p>Returns <code>FRAMEBUFFER_UNSUPPORTED</code> if the same color image is attached to more than one
```
Comment (vonture) [inline specs/latest/2.0/index.html]: Maybe "if the same image is attached to more than one color attachment point" instead?

File: specs/latest/2.0/index.html [hunk 2]
```diff
@@ -3308,6 +3310,22 @@ <h2 class="no-toc">Table of contents</h2>
         OpenGL ES 3.0.4 &sect;3.9.2</a>).
     </div>
 
+    <h3><a name="FRAMEBUFFER_COLOR_ATTACHMENTS">Framebuffer color attachments</a></h3>
+
+    <p>
+        If a color image is attached to more than one attachment point in a framebuffer,
+        <code>checkFramebufferStatus</code> returns <code>FRAMEBUFFER_UNSOPPORTED</code>.
+        (An color image can be an individual mip level, an array slice (from either 2D array
+         or cube map textures), or a 3D texture slice.)
```
Comment (vonture) [inline specs/latest/2.0/index.html]: textures->texture

File: specs/latest/2.0/index.html [hunk 3]
```diff
@@ -3308,6 +3310,22 @@ <h2 class="no-toc">Table of contents</h2>
         OpenGL ES 3.0.4 &sect;3.9.2</a>).
     </div>
 
+    <h3><a name="FRAMEBUFFER_COLOR_ATTACHMENTS">Framebuffer color attachments</a></h3>
+
+    <p>
+        If a color image is attached to more than one attachment point in a framebuffer,
+        <code>checkFramebufferStatus</code> returns <code>FRAMEBUFFER_UNSOPPORTED</code>.
+        (An color image can be an individual mip level, an array slice (from either 2D array
```
Comment (vonture) [inline specs/latest/2.0/index.html]: Would suggest "An color image" -> "An image"

Comment (vonture) [inline specs/latest/2.0/index.html]: I don't think it's necessary to wrap the whole sentence in parens.

File: specs/latest/2.0/index.html [hunk 4]
```diff
@@ -3308,6 +3310,22 @@ <h2 class="no-toc">Table of contents</h2>
...
+         or cube map textures), or a 3D texture slice.)
+    </p>
+
+    <div class="note rationale">
+        This is a limitation in Direct3D 11, on top of which OpenGL ES 3 behavior is emulated in Windows.
```
Comment (vonture) [inline specs/latest/2.0/index.html]: "limitation in"->"limitation of" and "emulated in"->"emulated on"

File: specs/latest/2.0/index.html [hunk 5]
```diff
@@ -3308,6 +3310,22 @@ <h2 class="no-toc">Table of contents</h2>
...
+        (see <a href="https://msdn.microsoft.com/en-us/library/windows/desktop/ff476464.aspx">msdn manual
+        page</a>). In order to have consistent WebGL 2 behaviors across platforms, it is reasonable to add
```
Comment (vonture) [inline specs/latest/2.0/index.html]: "behaviors"->"behavior"

File: specs/latest/2.0/index.html (hunk 1 above)
Comment (zhenyao) [inline specs/latest/2.0/index.html]: Done

File: specs/latest/2.0/index.html (hunk 3 above)
Comment (zhenyao) [inline specs/latest/2.0/index.html]: Done

File: specs/latest/2.0/index.html (hunk 4 above)
Comment (zhenyao) [inline specs/latest/2.0/index.html]: Done

File: specs/latest/2.0/index.html (hunk 5 above)
Comment (zhenyao) [inline specs/latest/2.0/index.html]: Done

**Comment by zhenyao** [comment]
Revised.

File: specs/latest/2.0/index.html [hunk 6]
```diff
@@ -3308,6 +3310,22 @@ <h2 class="no-toc">Table of contents</h2>
         OpenGL ES 3.0.4 &sect;3.9.2</a>).
     </div>
 
+    <h3><a name="FRAMEBUFFER_COLOR_ATTACHMENTS">Framebuffer color attachments</a></h3>
+
+    <p>
+        If a color image is attached to more than one attachment point in a framebuffer,
```
Comment (vonture) [inline specs/latest/2.0/index.html]: nit: two more instances of "color image"->"image" and "attachment point"->"color attachment point" in this paragraph.

**Comment by vonture** [comment]
LGTM with last nit.

File: specs/latest/2.0/index.html (hunk 6 above)
Comment (zhenyao) [inline specs/latest/2.0/index.html]: Done. Thanks.

**Comment by kenrussell** [comment]
LGTM.

**Pull request merged**